src/modules/counters.js

```diff
@@ -0,0 +1,7 @@
+const counters = () => {
+  const numOfCard = document.querySelector('.main, .items-counter').children.length;
```

Comment:
The .items-counter isn't doing anything here. We want to get the length children of the main element only